elasticsearch/experiment.yaml

```diff
@@ -101,23 +86,21 @@ spec:
         - name: cluster.name
           value: rally-demo
         - name: client.replicas
-          value: "{{ .Values.client_replicas }}"
+          value: "2"
```

Comment:
If we're not tuning replicas, should this just move from the experiment.yaml to the application manifests?